packages/components/src/HeaderBar/HeaderBar.component.js

```diff
@@ -271,13 +269,13 @@ if (process.env.NODE_ENV !== 'production') {
 	};
 
 	HeaderBar.propTypes = {
-		logo: PropTypes.shape(Logo.propTypes),
+		logo: PropTypes.shape({ ...Logo.propTypes, t: undefined }),
```

Comment:
I find weird to add t: undefined. _.omit(object, 't') will be better ?

Reply:
@jare-talend but we don't have lodash dependency in components ?

Reply:
Ah, i see it.